### DISCUSSION TITLE
LoRaWAN 1.0.4?

### DISCUSSION THREAD

**Discussion by tve** [jgromes/RadioLib#1276] [General]
I was hoping to use RadioLib for LoRaWAN but then discovered that Helium only supports v1.0.x, ugh. Is there interest in a version that does 1.0.4 or is that a non-starter?

**Comment by StevenCellist** [comment]
Look at the README in examples/LoRaWAN ;)

**Comment by tve** [comment]
I see, looks like I didn't remember that by the time I got to https://github.com/jgromes/RadioLib/blob/master/examples/LoRaWAN/LoRaWAN_Starter/notes.md?plain=1#L21 ("Helium does not support LoRaWAN v1.1 which is the version implemented by RadioLib")
Thanks for the correction!

**Reply by StevenCellist** [reply]
Fair shout, that phrasing pre-dates the support for 1.0.4. We'll change the wording a bit.
